deploy/materialized_view_flat.sql

```diff
@@ -4,34 +4,26 @@
 begin;
 
 create materialized view ggircs_swrs.flat as (
-  with x as (
-    select
-           ghgr_import.id as ghgr_import_id,
-           ghgr_import.xml_file as source_xml,
-           ghgr_import.imported_at
-    from ggircs_swrs.ghgr_import
-    order by ghgr_import_id desc
-  )
-  select ghgr_import_id,
+  select id as ghgr_import_id,
```

Comment:
creating an id column in the materialized view deploy file will conflict with the table exporter. The table exporter also creates an id column on each table when they are exported to the ggircs namespace. (Flat is not currently being exported to the ggircs namespace, but @hamzajaved says that there are plans to add it in the future)

Reply:
renaming `id` as `ghgr_import_id` ;)